static/js/compose.js

```diff
@@ -516,17 +510,6 @@ export function initialize() {
         subscriber_api.add_user_ids_to_stream([user_id], sub, success, xhr_failure);
     });
 
-    $("#compose_invite_users").on("click", ".compose_invite_close", (event) => {
-        const $invite_row = $(event.target).parents(".compose_invite_user");
-        const $all_invites = $("#compose_invite_users");
-
-        $invite_row.remove();
-
-        if ($all_invites.children().length === 0) {
-            $all_invites.hide();
-        }
-    });
```

Comment:
What did this code do -- and does the new implementation do this somewhere else?

Reply:
Previously the way it worked was there would be a parent div for each kind of banner (the ones I removed from `compose.hbs`) and banners would be put inside the relevant div when created.

Now the way it works is there is a single div that contains all banners.

This deleted code checks to see if there are no more warnings of this type, and hides the parent div if so. This isn't relevant to the new way it works because there are no parent divs that are show/hidden.

If that doesn't make sense I can try to clarify further.